include/openmc/dagmc.h

```diff
@@ -184,6 +188,11 @@ class DAGUniverse : public Universe {
                              //!< generate new material IDs for the universe
   bool has_graveyard_; //!< Indicates if the DAGMC geometry has a "graveyard"
                        //!< volume
+  std::map<int, vector<std::string>>
```

Comment:
Now that we're only writing IDs to the XML file can we make the values here `vector<int>`?